### PR TITLE
Cset tool updates

### LINKED ISSUE #2023: Support CSETv1 Publication
Engineering work to support CSET publication:

- Write a migration merging the CSETv1 annotator classifications such that...

  - If `CSETv1_Annotator-1` and `CSETv1_Annotator-2` agree,
    the matching value gets copied to `CSETv1`

  - If there's a missing and a non-missing field,
    use the non-missing one.

  - For notes fields, both annotators' texts 
    get concatenated with a textual label. E.g.
    ```
    Annotator 1: Lorem ipsum...

    Annotator 2: Twas brillig...
    ```
  - For array fields, the arrays get merged with duplicates removed.
  - For entities fields, the lists of entities get merged.

- Move the button to delete an entity into the entity header.

- In the comparison tool, color highlight rows
  where the annotators' values don't match.

### PR DESCRIPTION
**Pull request by cesarvarela**
closes #2023

### CODE REVIEW

File: site/gatsby-site/src/components/classifications/CsetTable.js [hunk 1]
```diff
@@ -40,12 +292,129 @@ export default function CsetTable({ data, className = '', ...props }) {
...
+      addToast({
+        message: 'Classification updated',
+        severity: SEVERITY.success,
+      });
+
+      setSubmitting(false);
+    } catch (error) {
+      addToast({
+        message: 'Classification updated',
+        severity: SEVERITY.danger,
```
Comment (lmcnulty) [inline site/gatsby-site/src/components/classifications/CsetTable.js:401]: I think the message should be different for a failure.

**Review by lmcnulty — COMMENTED** [review]
Some of the metadata fields are likely to change after merge. We should probably exclude them:

image


It seems like false + true is resolving to true:

image

---

I think we were planning to make this tool read-only and do edits in the  form. I'm a little concerned about the user making changes in both places at once, and then having data overwritten. But this looks good enough that I think we should talk with Mia or Heather about it again.

**Comment by cesarvarela** [comment]
> Some of the metadata fields are likely to change after merge. We should probably exclude them:

What do you mean by that, should I skip those listed in the first pic?

@lmcnulty

**Comment by lmcnulty** [comment]
> What do you mean by that, should I skip those listed in the first pic?

Yes. Sorry for being unclear.

**Comment by cesarvarela** [comment]
@lmcnulty @smcgregor updated, any last words before merging?

**Comment by smcgregor** [comment]
Looking great. We might want to have the entity delete on the merged records, but I think we should get this into prod and get feedback rather than make any more changes.

**Pull request merged**